app/views/provisioned.html

```diff
@@ -0,0 +1,156 @@
+<project-header class="top-header"></project-header>
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header header-toolbar">
+        <div class="container-fluid">
+          <div class="page-header page-header-bleed-right page-header-bleed-left">
+            <h1>
+              Provisioned Services
+              <span class="page-header-link">
+                <a ng-href="{{'services' | helpLink}}" target="_blank">
+                  Learn More <i class="fa fa-external-link" aria-hidden="true"></i>
+                </a>
+              </span>
+            </h1>
+          </div>
+          <div class="data-toolbar">
+            <div class="data-toolbar-filter">
+              <project-filter></project-filter>
+            </div>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content">
+        <div class="container-fluid">
+          <alerts alerts="alerts"></alerts>
+          <div class="row">
+            <div class="col-md-12">
+              <table class="table table-bordered table-hover table-mobile table-layout-fixed">
+                <colgroup>
+                  <col class="col-sm-3">
+                </colgroup>
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Bindings</th>
+                    <th>Status</th>
+                    <th>Created</th>
+                    <th>Web Interface Url</th>
+
+                    <!--<th>Name</th>
+                    <th>Cluster IP</th>
+                    <th>External IP</th>
+                    <th>Ports</th>
+                    <th>Selector</th>
+                    <th>Age</th>-->
+                  </tr>
+                </thead>
+                <tbody ng-if="(serviceInstances | hashSize) === 0">
+                  <tr>
+                    <td colspan="5"><em>{{emptyMessage}}</em></td>
+                  </tr>
+                </tbody>
+                <tbody ng-if="(serviceInstances | hashSize) > 0">
+                  <tr ng-repeat="serviceInstance in serviceInstances | orderObjectsByDate : true">
```

Comment:
Better to sort in the controller, not the view.

Use `track by (serviceInstance | uid)`